packages/atlas/src/views/global/YppDashboard/YppDashboard.tsx

```diff
@@ -0,0 +1,174 @@
+import BN from 'bn.js'
```

Comment:
Dashboard shouldn't be global, it should be only available in studio. For now, let's have YPP in viewer link to landing page and in studio, let's have it link to the dashboard, we'll update later